src/queries/card-id-to-pr-query.ts

```diff
@@ -1,21 +1,21 @@
-import { gql } from "@apollo/client/core";
+import { gql, TypedDocumentNode } from "@apollo/client/core";
 import { client } from "../graphql-client";
 import { PullRequestState } from "./schema/graphql-global-types";
-import { CardIdToPr } from "./schema/CardIdToPr";
+import { CardIdToPr, CardIdToPrVariables } from "./schema/CardIdToPr";
 
 interface CardPRInfo {
     number: number;
     state: PullRequestState;
 }
 
 export const runQueryToGetPRForCardId = async (id: string): Promise<CardPRInfo | undefined> => {
-    const info = await client.query<CardIdToPr>({
+    const info = await client.query({
         query: gql`
             query CardIdToPr($id: ID!) {
                 node(id: $id) {
                     ... on ProjectCard { content { ... on PullRequest { state number } } }
                 }
-            }`,
+            }` as TypedDocumentNode<CardIdToPr, CardIdToPrVariables>,
```

Comment:
Is this needed?  Everything looks fine when I drop it...

```suggestion
            }`,
```

Reply:
In that case e.g. `info.data.node` becomes `any` so it's fine, but less strict

Reply:
Ah, I missed that.